assets/js/components/product-attribute-control/index.js

```diff
@@ -89,7 +90,7 @@ class ProductAttributeControl extends Component {
 
 	render() {
 		const { list, loading } = this.state;
-		const { selected, onChange } = this.props;
+		const { onChange, onOperatorChange, operator = 'any', selected } = this.props;
```

Comment:
Should the operator default be set by `ProductAttributeControl.defaultProps` instead? That matches how we usually sets defaults in `wc-admin` (And I think if we ever run component documentation on this it picks that up as well?).